timesketch/lib/utils.py

```diff
@@ -259,11 +259,17 @@ def read_and_validate_jsonl(file_handle):
     # Fields that must be present in each entry of the JSONL file.
     mandatory_fields = ['message', 'datetime', 'timestamp_desc']
     lineno = 0
+    max_len_line_jsonl = current_app.config.get('MAX_LENGTH_LINE_JSONL')
```

Comment:
Can you add a default value here in case the config variable is not set (which will happen in a lot of cases where people do not update their config file to match the recent version)?

Reply:
Done